pkg/executor/importer/kv_encode.go

```diff
@@ -62,8 +62,6 @@ func NewTableKVEncoder(
 	if err != nil {
 		return nil, err
 	}
-	// we need a non-nil TxnCtx to avoid panic when evaluating set clause
-	baseKVEncoder.SessionCtx.SetTxnCtxNotNil()
```

Comment:
This is not needed any more because we have removed `SessionVars` in the new session.